src/ir/context.rs

```diff
@@ -697,6 +702,19 @@ impl<'ctx> BindgenContext<'ctx> {
             CXType_Float => TypeKind::Float(FloatKind::Float),
             CXType_Double => TypeKind::Float(FloatKind::Double),
             CXType_LongDouble => TypeKind::Float(FloatKind::LongDouble),
+            CXType_Complex => {
+                let float_type = ty.elem_type()
+                    .expect("Not able to resolve complex type?");
+                let float_kind = match float_type.kind() {
+                    CXType_Float => FloatKind::Float,
+                    CXType_Double => FloatKind::Double,
+                    CXType_LongDouble => FloatKind::LongDouble,
+                    _ => panic!("Non floating-type complex?"),
+                };
+
+                self.saw_complex.set(true);
```

Comment:
That makes a lot of sense.
Should I add a non white listed _Complex in whitelist_basic.hpp, or a new whitelist_complex.h?